trackersite/tracker/views.py

```diff
@@ -556,19 +666,629 @@ def cluster_detail(request, pk):
+            elif giveexample == 'topic':
+                response = HttpResponseCsv(['name', 'grant', 'new_tickets', 'media', 'preexpenses', 'expenses', 'description', 'form_description'])
+                response['Content-Disposition'] = 'attachment; filename="example-topic.csv"'
+                response.writerow([u'Jméno tématu', u'Název grantu', u'True/False', u'True/False', u'True/False', u'True/False', u'Popis tématu', u'Popis formuláře tématu'])
+                return response
+            elif giveexample == 'grant':
+                response = HttpResponseCsv(['full_name', 'short_name', 'slug', 'description'])
+                response['Content-Disposition'] = 'attachment; filename="example-grant.csv"'
+                response.writerow([u'Plné jméno', u'Krátké jméno', u'Slug', u'Popis'])
+                return response
+            elif giveexample == 'expense':
+                    fields = ['ticket_id', 'description', 'amount', 'wage']
+                    if request.user.is_staff:
+                        fields.append('accounting_info')
+                        fields.append('paid')
+                    response = HttpResponseCsv(fields)
+                    response['Content-Disposition'] = 'attachment; filename="example-expense.csv"'
+                    row = [u'ID tiketu', u'Popis výdaje', u'Vydané peníze v CZK', u'True/False']
+                    if request.user.is_staff:
+                        row.append(u'Účetní údaje')
+                        row.append(u'True/False')
+                    response.writerow(row)
+                    return response
+            elif giveexample == 'preexpense':
+                response = HttpResponseCsv(['ticket_id', 'description', 'amount', 'wage'])
+                response['Content-Disposition'] = 'attachment; filename="example-preexpense.csv"'
+                response.writerow([u'ID tiketu', u'Popis výdaje', u'Vydané peníze v CZK', u'True/False'])
+                return response
+            elif giveexample == 'user':
+                response = HttpResponseCsv(['username', 'password', 'first_name', 'last_name', 'is_superuser', 'is_staff', 'is_active', 'email'])
+                response['Content-Disposition'] = 'attachment; filename="example-user.csv"'
+                response.writerow([u'Uživatelské jméno', u'Heslo', u'KřestníJméno', u'Příjmení', u'True/False', u'True/False', u'True/False', u'emailova@adresa.cz'])
+                return response
+            elif giveexample == 'media':
+                response = HttpResponseCsv(['ticket_id', 'url', 'description', 'number'])
+                response['Content-Disposition'] = 'attachment; filename="example-media.csv"'
+                response.writerow([u'1', u'http://odkaz.cz', u'popisek', u'100'])
+                return response
+            else:
+                return HttpResponseBadRequest("You can't want example file of invalid object")
+        else:
+            return render(request, 'tracker/import.html', {})
```

Comment:
Jako výše. Takový kód se nedá rozumně upravovat.

Reply:
Pokusím se zjednodušit.